core/tests/protractor_utils/AdminPage.js

```diff
@@ -292,6 +333,154 @@ var AdminPage = function() {
     }
   };
 
+  this.getMiscTab = async function() {
+    await action.click('Misc Tab Button', miscTabButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.uploadTopicSimilarities =
+    async function(relativePathToSimilaritiesFile, isValidFile) {
+      var absPath = path.resolve(__dirname, relativePathToSimilaritiesFile);
+      await waitFor.visibilityOf(chooseSimilarityFileInput,
+        'Similarity upload form is not visible');
+      await chooseSimilarityFileInput.sendKeys(absPath);
+      await action.click('Upload button', similarityFileUploadButton);
+      if (isValidFile) {
+        var text = 'Topic similarities uploaded successfully.';
+        await waitFor.visibilityOf(statusMessage,
+          'Status message not visible');
+        await waitFor.textToBePresentInElement(statusMessage, text,
+          'Text not showing up in status message');
+      } else {
+        var text = 'Server error: \'ascii\' codec can\'t encode characters' +
+          ' in position 1024-1025: ordinal not in range(128)';
+        await waitFor.visibilityOf(statusMessage,
+          'Status message not visible');
+        await waitFor.textToBePresentInElement(statusMessage, text,
+          'Text not showing up in status message');
+      }
+    };
+
+  this.expectSimilaritiesToBeUploaded = async function() {
+    await waitFor.elementToBeClickable(similarityDownloadButton,
+      'Similarity upload failed – download similarities button not clickable.');
+  };
+
+  this.downloadSimilarityFile = async function() {
+    await action.click('Download Similarity File Button',
+      similarityDownloadButton);
+  };
+
+  this.clearSearchIndex = async function() {
+    await action.click('Clear search index button', searchIndexClearButton);
+    await general.acceptAlert();
+    await waitFor.visibilityOf(statusMessage,
+      'Status message not showing up.');
+    await waitFor.textToBePresentInElement(statusMessage,
+      'Index successfully cleared.');
+  };
+
+  this.flushMigrationBotContributions = async function() {
+    await action.click('Migration bot flush contributions button',
+      flushMigrationBotContributionsButton);
+    await general.acceptAlert();
+    await waitFor.textToBePresentInElement(statusMessage,
+      'Migration bot contributions successfully flushed.',
+      'Migration bot cuntributions not flushing.');
```

Comment:
'Succes message for migration bot contributions flushing not visible.' (something around this!)